zipline/_protocol.pyx

```diff
@@ -76,6 +81,289 @@ cdef class BarData:
             self.data_frequency
         )
 
+    def spot_value(self, assets, fields):
+        """
+        Returns the spot value of the given assets for the given fields
+        at the current simulation time.  Spot values are the as-traded price
```

Comment:
Based on the comment below on `price`, it seems like this isn't quite true anymore?